deploy/helm/observability/otel-collector/values.yaml

```diff
@@ -141,6 +141,14 @@ collector:
           exporters:
             - debug
             - otlphttp/dev
+        metrics:
```

Comment:
This config (`metrics`) allows `otel-collector` to accept metrics info from our apps (on top of tracing info).